userprofile/templates/userprofile/profile.html

```diff
@@ -12,62 +12,60 @@
 {% endblock %}
 
 {% block content %}
-<div class="page-header">
-    <h1>
-        {% if user.first_name or user.last_name %}
-        {{ user.first_name }} {{ user.last_name }}
-        {% else %}
-        {% trans "[Unknown name]" %}
-        {% endif %}
-    </h1>
+<div class="row">
+	<div class="col-md-offset-2 col-md-2 page-header">
+		<div class="gravatar">
+			<a href="http://gravatar.com" target="_blank" rel="noopener noreferrer">
+				<img class="img-circle" src="{{ profile.avatar_url }}" alt="" aria-label="Change your avatar">
+				<span class="notify-badge">Change</span>
+			</a>
+		</div>
+		<h1>
+			{% if user.first_name or user.last_name %}
+			{{ user.first_name }} {{ user.last_name }}
+			{% else %}
+			{% trans "[Unknown name]" %}
+			{% endif %}
+		</h1>
+		<h3 style="margin-top: -10px;" class="text-muted">{% firstof user.username miss %}</h3>
+	</div>
 </div>
 <div class="row">
-    <div class="col-md-2">
-        <div class="thumbnail">
-            <img src="{{ profile.avatar_url }}" alt="avatar" />
-        </div>
-        <dl>
-            {% trans "[Missing]" as miss %}
-        	  <dt>{% trans "Student ID" %}</dt>
+    <div class="col-md-offset-2 col-xs-10 col-md-3">
+		<h2>Personal Information:</h2>
```

Comment:
The issue #590 already had a reason for separating the user data stored in A+ and the data from the identity provider. I prefer that approach.

Reply:
I read the conversation, but I did not see the benefits of separating the data stored in A+ from the data that is comming from the  identity provider. Why do we want to do that?. Besides, the amount of data that is shown in this webpage is not really huge, and having one header for showing only two values does not seem practical. It  is more practical two have one header and four values below.

Reply:
The language is a user preference set in A+ and API tokens are a feature in A+. We could have more API tokens in the future and the user should also be able to delete and regenerate them. The list of automatic LTI redirections is also specific to A+. Then there are user fields that originate from the identity provider and the user can not change them in A+. Using different sections emphasizes the difference for accessibility and usability. We may also add new A+-specific settings in the future.